exporters/requirements-dev.txt

```diff
@@ -1,6 +1,9 @@
 # Needed by exporters/tests
 pytest
 
+# Integration Tests
+httpie
+
```

Comment:
We actually don't need this. I was initially using it because httpie had nicer timeout/retry flags than curl, but then forgot to use it myself 😅 

Looks like you've handled that in the script anyway.

Reply:
ok removed.